db/database.go

```diff
@@ -129,6 +129,7 @@ type DatabaseContext struct {
 	singleCollection             *DatabaseCollection            // Temporary collection
 	CollectionByID               map[uint32]*DatabaseCollection // A map keyed by collection ID to Collection
 	CollectionNames              map[string]map[string]struct{} // Map of scope, collection names
+	ClusterUUID                  string                         // uuid of cluster
```

Comment:
Do we have some duplication between ClusterUUID/getClusterUUID and serverUUID/GetServerUUID, or am I misunderstanding the difference?

Reply:
just a mistake, settled on ServerUUID as we use it elsewhere.